src/citrine/informatics/predictors.py

```diff
@@ -272,3 +276,122 @@ def __str__(self):
     def post_build(self, project_id: UUID, data: dict):
         """Creates the predictor report object."""
         self.report = ReportResource(project_id, self.session).get(data['id'])
+
+
+class MolecularStructureFeaturizer(Serializable['MolecularStructureFeaturizer'], Predictor):
+    """
+    [ALPHA] A "batteries-included" featurizer for organic molecules. Powered by CDK.
+
+    Parameters
+    ----------
+    name: str
+        name of the configuration
+    description: str
+        the description of the predictor
+    descriptor: MolecularStructureDescriptor
+        the descriptor to featurize
+    features: List[str]
+        the list of features to compute. Valid values include:
+        - all
+        - standard
+        - expensive
+        - PetitjeanShapeIndex
+        - TopologicalSurfaceArea
+        - EccentricConnectivityIndex
+        - MomentOfInertia
+        - KappaShapeIndices
+        - RuleOfFive
+        - ALOGP
+        - PolarizabilityAutocorr
+        - BondCount
+        - Volume
+        - VertexAdjacencyMagnitude
+        - ZagrebIndex
+        - HBondDonorCount
+        - ChiPathCluster
+        - MannholdLogP
+        - AtomCount
+        - AutocorrelationDescriptorCharge
+        - AromaticAtomCount
+        - LargestPiSystem
+        - HybridizationRatioDescriptor
+        - LargestChain
+        - MDE
+        - Weight
+        - LengthOverBreadth
+        - FMF
+        - AtomicPolarizability
+        - CarbonTypes
+        - PetitjeanNumber
+        - HBondAcceptorCount
+        - RotatableBondsCount
+        - BCUT
+        - WeightedPath
+        - AromaticBondCount
+        - GravitationalIndex
+        - XLogP
+        - ChiPath
+        - WHIM
+        - FragmentComplexity
+        - ChiChain
+        - KierHallSmarts
+        - MassAutocorr
+        - AcidGroupCount
+        - BPol
+        - WienerNumbers
+        - LongestAliphaticChain
+        - CPSA
+        - ChiCluster
+        - BasicGroupCount
+    excludes: List[str]
+        list of features to exclude (accepts same set of values as features)
```

Comment:
Document what happens with values in both lists?

Reply:
You mean explain that excludes means "any feature in this list and also the features list will be excluded"?

Reply:
I think that's what he means.

I got asked this question, what happens if a feature is in both lists, when writing up the design doc. So I do think we need to explicitly explain the pattern "Use 'features' to specify a superset of features and use 'exclude' to whittle it down."